migrations/20200605172945_add_commit_score_column.ts

```diff
@@ -0,0 +1,13 @@
+import * as Knex from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table("commit", (builder) => {
+    builder.integer("score").notNullable();
```

Comment:
When adding non-nullable column, the best practice is either adding the column as non-nullable with default value, or:
- add the column as nullable
- update all rows with a default value in the column
- alter the column to be non-nullable.

Otherwise, DB may fail with error if there are existing rows.